src/org/rust/lang/codestyle/RustCodeStyleSettings.kt

```diff
@@ -0,0 +1,7 @@
+package org.rust.lang.codestyle
+
+import com.intellij.psi.codeStyle.CodeStyleSettings
+import com.intellij.psi.codeStyle.CustomCodeStyleSettings
+
+class RustCodeStyleSettings constructor(container: CodeStyleSettings) :
```

Comment:
seems like that is indeed not necessary. I think it was created by a Java-to-Kotlin conversion.